src/components/vuestic-components/va-content/VaContent.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="content">
```

Comment:
We want `.content` class and `va-content` component.
Those should be handled separately. You can use mixin or extent to abstract away class, but `.content` definitely doesn't belong to va-content.

Also naming. VaContent component should have class `va-content`.